src/App.tsx

```diff
@@ -4,3 +1,7 @@
+import { Routes, Route, Navigate, NavLink } from 'react-router-dom';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
```

Comment:
Verify that the import path for `NotFoundPage` is correct. It is imported from `./pages/NotFoundPage`, which should match your project structure.